sky/provision/aws/instance.py

```diff
@@ -46,24 +46,10 @@
 # https://aws.amazon.com/ec2/pricing/on-demand/#Data_Transfer_within_the_same_AWS_Region
 
 
-@aws.import_package
-def botocore_config():
-    """Get botocore.config module.
-
-    This is for backward compatibility when `aws` module is imported before the
-    `botocore_config()` function is added in #1702.
-    TODO: remove this function after 0.6.0 release, when new provisioner is
-    used by default.
-    """
-    config = aws.botocore.config
-    return config
-
-
 def _default_ec2_resource(region: str) -> Any:
```

Comment:
I wonder if we can come up with an easier repro. Import this module and repeatedly calls `_default_ec2_resource('us-east-1')`?

Reply:
Added a unit test for it in `tests/unit_test/clouds/test_aws.py`